sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/Sonar.java

```diff
@@ -21,8 +21,15 @@
 
 public class Sonar extends Artifact {
```

Comment:
Line 63, we are handling a tag name "LTS". I think we should introduce a tag "LTA" as a replacement, WDYT?